agent/xds/envoy_versioning.go

```diff
@@ -13,8 +13,7 @@ var (
 	// the zero'th point release of the last element of proxysupport.EnvoyVersions.
 	minSupportedVersion = version.Must(version.NewVersion("1.17.0"))
 
-	// add min version constraints for associated feature flags when necessary, for example:
-	// minVersionAllowingEmptyGatewayClustersWithIncrementalXDS = version.Must(version.NewVersion("1.16.0"))
+	minVersionWithLDSAndCDSProperlyUsingWildcardsWithIncrementalXDS = version.Must(version.NewVersion("1.19.0"))
```

Comment:
nit: could we somehow make this var shorter and add more detail in the comment?

Reply:
Historically I'd gone with horribly awful names deliberately because this is code meant to be erased in 1-2 consul releases when the relevant envoy versions roll away out of the support matrix.

Reply:
I aligned the naming here with the feature struct name.